task09/task09.1/task09.1/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task09._1
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введение значение x");
+            double x = double.Parse(Console.ReadLine());
+            Console.WriteLine("Введите значение n");
+            int n = int.Parse(Console.ReadLine());
+            double Sum = 0;
```

Comment:
Называйте локальные переменные с маленькой буквы.
```suggestion
            double sum = 0;
```